exchanges/bybit/bybit_wrapper.go

```diff
@@ -285,123 +299,84 @@ func (by *Bybit) Run(ctx context.Context) {
 
 // FetchTradablePairs returns a list of the exchanges tradable pairs
 func (by *Bybit) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.Pairs, error) {
-	if !by.SupportsAsset(a) {
+	var pair currency.Pair
+	var category string
+	switch a {
+	case asset.Spot, asset.Options, asset.CoinMarginedFutures, asset.USDCMarginedFutures, asset.USDTMarginedFutures:
+		category = getCategoryName(a)
+	default:
 		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, a)
 	}
-
-	var pair currency.Pair
+	allPairs, err := by.GetInstruments(ctx, category, "", "Trading", "", "", 0)
+	if err != nil {
+		return nil, err
+	}
+	pairs := make([]currency.Pair, 0, len(allPairs.List))
 	switch a {
-	case asset.Spot:
-		allPairs, err := by.GetAllSpotPairs(ctx)
-		if err != nil {
-			return nil, err
-		}
-		pairs := make([]currency.Pair, len(allPairs))
-		for x := range allPairs {
-			pair, err = currency.NewPairFromStrings(allPairs[x].BaseCurrency,
-				allPairs[x].QuoteCurrency)
-			if err != nil {
-				return nil, err
-			}
-			pairs[x] = pair
-		}
-		return pairs, nil
-	case asset.CoinMarginedFutures:
-		allPairs, err := by.GetSymbolsInfo(ctx)
-		if err != nil {
-			return nil, err
-		}
-		pairs := make([]currency.Pair, 0, len(allPairs))
-		for x := range allPairs {
-			if allPairs[x].Status != "Trading" || allPairs[x].QuoteCurrency != "USD" {
-				continue
-			}
-
-			contractSplit := strings.Split(allPairs[x].Name, allPairs[x].BaseCurrency)
-			if len(contractSplit) != 2 {
-				log.Warnf(log.ExchangeSys, "%s base currency %s cannot split contract name %s cannot add to tradable pairs",
-					by.Name,
-					allPairs[x].BaseCurrency,
-					allPairs[x].Name)
-				continue
-			}
-
-			pair, err = currency.NewPairFromStrings(allPairs[x].BaseCurrency,
-				contractSplit[1])
+	case asset.Spot, asset.Options:
+		for x := range allPairs.List {
+			pair, err = currency.NewPairFromString(allPairs.List[x].Symbol)
```

Comment:
Like the other times, new pair from string on its own has its failing:
<img width="1296" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9261323/e7271c53-60e1-4ad4-9422-a1bc49e9bc2b">
Every pair in that screenshot is incorrect. Please update and verify that you are handling currency pairs properly